README.md

```diff
@@ -0,0 +1,182 @@
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+<details>
+  <summary>📗 Table of Contents</summary>
+  <ul>
+    <li><a href="#-budget-app-">📖 Budget App</a></li>
+    <li><a href="#-built-with-">🛠 Built With</a></li>
+    <li><a href="#-getting-started-">💻 Getting Started</a></li>
+    <li><a href="#-authors-">👥 Authors </a></li>
+    <li><a href="#-future-features-">🔭 Future Features</a></li>
+    <li><a href="#-contributing-">🤝 Contributing</a></li>
+    <li><a href="#️-show-your-support-">⭐️ Show your support </a></li>
+    <li><a href="#-acknowledgments-">🙏 Acknowledgments </a></li>
+    <li><a href="#-license-">📝 License</a></li>
+  </ul>
+</details>
+
+<br>
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Budget App <a name="about-project"></a>
+
+**Budget App** is a Ruby on Rails project designed to help you monitor you spending habits. Go ahead and sign up to start organizing your transactions, monitoring your total spend and keeping track of your budget.
+![image](https://github.com/Booze33/budget-app/assets/99689722/89631a5b-54ea-419d-8c7c-48b12701dc76)
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">HTML</a></li>
+    <li><a href="#">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://rubyonrails.org/">Rails</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- User authentication
+- Users can create and view transaction categories
+- Users can create and view transactions
+- Users can monitor total spend on each category
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://budgetapp-v6n9.onrender.com)
+- [Video Documentation](https://www.loom.com/share/e8b880927021464c85eea4e8813099ca?sid=c02f61e8-a033-4c45-bf32-ff4ef1ddc891)
```

Comment:
- [ ] [OPTIONAL] Great presentation @Booze33 , however, I would suggest you do a video demonstration of the mobile version next time since this project is about the mobile version of this web app.